src/htmlaudio/HTMLAudioInstance.ts

```diff
@@ -469,6 +509,9 @@ export class HTMLAudioInstance extends PIXI.utils.EventEmitter implements IMedia
         this._pausedReal = false;
         this._paused = false;
         this._muted = false;
+        this._wait = 0;
+        this._waitStart = 0;
+        this._waitTimer = 0;
```

Comment:
Can you make sure that the waitTimer is cleared here? In case this is destroyed between the user play and the actual play starting.

Reply:
Good catch - I'd put that line into my editor, but forgot to hit save

Reply:
One quick follow up: does the clearTimeout spec support zero value here? Or is it better to wrap in an if. I saw mdn doc that passed undefined.

Reply:
clearTimeout doesn't throw any errors if passed the number of a non-valid timer (like one that's already been cleared), no value at all, or even non-numeric values. I'm fine with adding a check for a value here before calling it if you'd prefer.

Reply:
Yeah, lets do a check to be safe. I’d trust not all browsers got this correct.

Reply:
Awesome, thanks @ericente. I'll working on making a release of this ASAP